packages/danbooru/package.json

```diff
@@ -49,6 +49,6 @@
     "koishi-plugin-booru": "^1.0.3"
```

Comment:
You should change this line to be `1.1.0` either.

Reply:
oh, yes